### PR TITLE
Legacy QT removal

### PR DESCRIPTION
**Pull request by dgault**
This is the follow up to https://github.com/ome/bioformats/pull/4031

Removes LegacyQT Reader, Writer and Tools
This also impacts the PictReader and the Plugins options

I have opted to remove the QTWidgets and PictWidgets from the plugin as they will no longer have any options.

### CODE REVIEW

File: components/formats-bsd/src/loci/formats/in/QTReader.java [hunk 1]
```diff
@@ -32,29 +32,648 @@
...
+import loci.formats.codec.QTRLECodec;
+import loci.formats.codec.RPZACodec;
+import loci.formats.codec.ZlibCodec;
+import loci.formats.meta.MetadataStore;
 
 /**
- * QTReader is the file format reader for QuickTime movie files.
- * It does not read files directly, but chooses which QuickTime reader is
- * more appropriate.
+ * NativeQTReader is the file format reader for QuickTime movie files.
```
Comment (melissalinkert) [inline components/formats-bsd/src/loci/formats/in/QTReader.java]: `QTReader`?

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [BIOFORMATS-push#43](https://snoopycrimecop-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-push/43/). See the [console output](https://snoopycrimecop-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-push/43/consoleText) for more details.
Possible conflicts:
  - PR #4054 dgault 'Legacy ND2  removal'
    - components/bio-formats-plugins/src/loci/plugins/util/LociPrefs.java

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [BIOFORMATS-push#39](https://devspace_km-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-push/39/). See the [console output](https://devspace_km-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-push/39/consoleText) for more details.
Possible conflicts:
  - PR #4054 dgault 'Legacy ND2  removal'
    - components/bio-formats-plugins/src/loci/plugins/util/LociPrefs.java

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [BIOFORMATS-push#551](https://snoopycrimecop-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-push/551/). See the [console output](https://snoopycrimecop-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-push/551/consoleText) for more details.
Possible conflicts:
  - PR #4054 dgault 'Legacy ND2  removal'
    - components/bio-formats-plugins/src/loci/plugins/util/LociPrefs.java

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [BIOFORMATS-push#44](https://snoopycrimecop-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-push/44/). See the [console output](https://snoopycrimecop-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-push/44/consoleText) for more details.
Possible conflicts:
  - PR #4054 dgault 'Legacy ND2  removal'
    - components/bio-formats-plugins/src/loci/plugins/util/LociPrefs.java

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [BIOFORMATS-push#40](https://devspace_km-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-push/40/). See the [console output](https://devspace_km-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-push/40/consoleText) for more details.
Possible conflicts:
  - PR #4054 dgault 'Legacy ND2  removal'
    - components/bio-formats-plugins/src/loci/plugins/util/LociPrefs.java

~~--conflicts~~ Conflict resolved in build [BIOFORMATS-push#41](https://devspace_km-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-push/41/). See the [console output](https://devspace_km-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-push/41/consoleText) for more details.

**Review by melissalinkert — APPROVED** [review]
Looks good to me, and confirmed that `git log --follow -p QTReader.java` behaves as expected.

**Review by sbesson — APPROVED** [review]

**Pull request merged**